tests/test_decode.py

```diff
@@ -53,7 +53,7 @@ def get_test_linelocation_1():
     "Return a prepared line location with 3 LRPs"
     # References node 0 / line 1 / lines 1, 3
     lrp1 = LocationReferencePoint(13.41, 52.525,
-                                  FRC.FRC0, FOW.SINGLE_CARRIAGEWAY, 0.75,
+                                  FRC.FRC0, FOW.SINGLE_CARRIAGEWAY, 90/22,
```

Comment:
I think this should be `11.25` rather than `22`.

Reply:
Oops, totally right.